src/components/list/CategoryMenu.tsx

```
@@ -4,65 +4,65 @@ import { useState, useEffect } from "react"
+        <div className="max-w-7xl w-full mx-0 py-4">
+        {isLoading ?
+            <Spinner message="카테고리를 불러오는 중입니다..." />
+        : (
         <div className="grid grid-cols-5 sm:grid-cols-6 md:grid-cols-8 lg:grid-cols-10 gap-4 justify-items-center">
-            {categories.map((category) => (
             <div
+                key={0}
+                className={clsx("w-full flex flex-col items-center cursor-pointer p-3 border-b-2 border-gray-border rounded-lg rounded-lg w-23 hover:bg-gray-50", categoryId === 0 && "bg-secondary-color border-main-color")}
+                onClick={() => handleCategoryClick(0)}
+            >
+                <span className={clsx("text-sm text-center text-gray-500", categoryId === 0 && "text-main-color")}>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)